src/java/org/apache/cassandra/index/sai/disk/v2/hnsw/ExtendedHnswGraph.java

```diff
@@ -60,4 +61,25 @@ static int[] getSortedNodes(HnswGraph hnsw, int level) {
         Arrays.sort(sortedNodes);
         return sortedNodes;
     }
+
+    public static int[] getSortedNodes(ExtendedHnswGraph hnsw, int level, Function<Integer, Integer> ordinalsMapper)
```

Comment:
I don't think we need this in diskann, the hnsw stuff is only used for reading old files